<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>throbdecoder</string>
    </property>
   </widget>
   <widget class="QwtPlot" name="throbscope">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="s2n_display">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="metrics_display">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="throbTextbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modeSelector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>85</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>throb1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>throb2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>throb4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>throbx1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>throbx2</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="reverseSwitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>normal</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="throb_ifMarker">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="afcSwitch_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>afc off</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threshold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
